### DISCUSSION TITLE
How can I  write a authentication/authorization providers like Google, Facebook, Twitter, GitHub, etc.?

### DISCUSSION THREAD

**Discussion by azataiot** [fastapi/fastapi#7598] [Questions]
### First check

* [*] I used the GitHub search to find a similar issue and didn't find it.
* [*] I searched the FastAPI documentation, with the integrated search.
* [*] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I  write a authentication/authorization providers like Google, Facebook, Twitter, GitHub, etc.?

**Comment by s0h3ck** [comment]
@yaakovazat Have you checked the issue https://github.com/tiangolo/fastapi/issues/12?
